src/listen/websocket.rs

```diff
@@ -238,12 +257,60 @@ fn options_to_query_string(options: &Options) -> String {
     serde_urlencoded::to_string(serialized_options).unwrap_or_default()
 }
 
+#[derive(Debug)]
+pub struct TranscriptionStream {
+    write_arc: Arc<Mutex<SplitSink<WebSocketStream<MaybeTlsStream<TcpStream>>, Message>>>,
+}
+
+impl TranscriptionStream {
+    pub async fn finalize(
+        &self,
+        event_tx: Sender<Event>,
```

Comment:
`TranscriptionStream` should hold on to a `Sender<Event>` so the user doesn't have to pass it in to every message.

Reply:
Resolved